app/controllers/categories_controller.rb

```diff
@@ -47,4 +48,10 @@ def category_params
   def category_find
     @category = Category.find_by(id: params[:id])
   end
+
+  def for_admin
+    return unless current_user.admin_type == false
```

Comment:
You do not need  == false. You can write unless current_user.admin_type or unless current_user.admin_type?

Reply:
Updated